apps/gro/steps.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
     next: '/about'
   },
   '/about': {
+    template: 'step',
```

Comment:
yes it will be - would need a change to hof-controllers/wizard though - on my todo list